docs/resources.md

```diff
@@ -1,6 +1,6 @@
 # PipelineResources
 
-`PipelinesResources` in a pipeline are the set of objects that are going to be
+`PipelineResources` in a pipeline are the set of objects that are going to be
```

Comment:
nit `PipelineResource`s maybe ? :stuck_out_tongue: (although it doesn't seem to render super well..)